rest/route/aws.go

```diff
@@ -184,9 +184,10 @@ func (aws *awsSns) handleInstanceInterruptionWarning(ctx context.Context, instan
 	}
 
 	grip.Info(message.Fields{
-		"message": "got interruption warning from AWS",
-		"distro":  h.Distro,
-		"host_id": h.Id,
+		"message":       "got interruption warning from AWS",
+		"distro":        h.Distro.Id,
+		"instance_type": h.InstanceType,
```

Comment:
I think this field is never set for spot hosts that are started with the ec2_fleet type (which is most of them).
It would need to be pulled out of the host's distro.